qubes/events.py

```diff
@@ -115,6 +115,17 @@ def add_handler(cls, event, func):
         # pylint: disable=no-member
         cls.__handlers__[event].add(func)
 
+    @classmethod
+    def remove_handler(cls, event, func):
+        '''Remove event handler from subject's class.
+
+        :param str event: event identificator
+        :param collections.Callable handler: handler callable
+        '''
+
+        # pylint: disable=no-member
+        cls.__handlers__[event].remove(func)
+
```

Comment:
Still, I don't see any reason why duplicate event handling code when we have it in place already. As explained above - behavior of _invalid call_ to `remove_handler` does not warrant not implementing it.